Gopkg.toml

```diff
@@ -24,11 +24,6 @@
 #   go-tests = true
 #   unused-packages = true
 
-
-[[constraint]]
-  name = "github.com/coreos/bbolt"
-  version = "1.3.2"
-
```

Comment:
Add new constraint for go.etcd.io/bbolt

Reply:
word, do you want this pinned to a version?

Reply:
yes